### PR TITLE
BTCA-360 VBT Analytics

### PR DESCRIPTION
**Pull request by ManasAgarwalVlocity**

### CODE REVIEW

File: lib/vlocity.js [hunk 1]
```diff
@@ -125,8 +125,12 @@ Vlocity.runDataPacksCommand = async function(action, options) {
 
             process.chdir(options.chdir);
         }
+        var passedInOptions = JSON.parse(JSON.stringify(options)),
+            vlocity_run = new Vlocity(passedInOptions);
```
Comment (rutlabaga) [inline lib/vlocity.js]: Definitely this should not be moved like this.

Comment (rutlabaga) [inline lib/vlocity.js]: Definitely this new Vlocity should not be moved here.

Comment (rutlabaga) [reply lib/vlocity.js]: All the ones in this file are for VBT and can be ignored anyway.

Comment (ManasAgarwalVlocity) [reply lib/vlocity.js]: Changes have been made in vlocityCli.js to track actual command provided by user.  Now UI changes to send config variable(disableVbtAnalytics) can be ignored.

File: lib/vlocitycli.js [hunk 2]
```diff
@@ -779,3 +785,12 @@ VlocityCLI.prototype.runTestJob = async function(dataPacksJobData) {
     
     return { message: message, status: 'success', action: 'Test Job' };
 }
+
+VlocityCLI.prototype.updateAnalytics = function (eventAction) {
```
Comment (rutlabaga) [inline lib/vlocitycli.js]: There still needs to be a way to disable the analytics from the jobInfo and a command line argument.

**Pull request closed without merging**